packages/preview/zhongwenshu/0.0.1/typst.toml

```diff
@@ -0,0 +1,10 @@
+[package]
+name = "zhongwenshu"

```

Comment:
If I managed to translate this correctly, this means `chinese book` right? We try to avoid too general names in order to prevent an unfair advantage when discovering packages. See our [naming rules](https://github.com/typst/packages/blob/main/docs/manifest.md#naming-rules)